dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/util/AnalyticsUtils.java

```diff
@@ -222,7 +221,7 @@ else if ( decimals != null && decimals > 0 )
         }
         else
         {
-            return MathUtils.getRounded( value );
+            return new Integer( (int) Precision.round( value, 0 ) );
```

Comment:
Would be better to use Long instead of Integer, as double is 8 bytes and integer is 4 bytes, and we might get an exception from the conversion.